definitions/npm/ramda_v0.x.x/flow_v0.62.x-/ramda_v0.x.x.js

```diff
@@ -1845,14 +1845,13 @@ declare module ramda {
 
   declare function memoize<A, B, T: (...args: Array<A>) => B>(fn: T): T;
 
-  declare function memoizeWith<A, B>(
-    keyFn: (x: A) => string,
-    ...rest: Array<void>
-  ): (fn: (x: B) => B) => A;
-  declare function memoizeWith<A, B>(
-    keyFn: (x: A) => string,
-    fn: (x: B) => B
-  ): A;
+  declare function memoizeWith<A, B, C>(
+    keyFn: (...args: Array<A>) => C
```

Comment:
keyFn doesn't take an array of args AFAICT, can you point to docs otherwise?

Reply:
It does not. It takes a number of arguments. Notice the ... rest operator